tests/test.py

```diff
@@ -153,24 +170,37 @@ def wait_unix_socket(socket_path, count=10, sleep=0.1):
         time.sleep(sleep)
     raise Exception("can't wait connection")
 
+def get_unix_connection_string(port, dbname):
+    if TEST_MYSQL:
+        return get_postgresql_tcp_connection_string(port, dbname)
+    else:
+        return get_postgresql_unix_connection_string(port, dbname)
 
 def get_postgresql_unix_connection_string(port, dbname):
```

Comment:
i would rename these functions without `postgesql` word

Reply:
here I explicitly used this methods to show that for mysql used tcp connection. in a future I will add (if need) get_mysql_unix_connection_string (if it will different for mysql) and replace with it